tools/appsre-build-worker-packer.sh

```diff
@@ -7,6 +7,21 @@ COMMIT_SHA=$(git rev-parse HEAD)
 COMMIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
 ON_JENKINS=true
 SKIP_CREATE_AMI=false
+AMI_X86=ami-06f1e6f8b3457ae7c
+AMI_AARCH64=ami-06bbacb93891d7356
+INST_TYPE_X86=c5.large
+INST_TYPE_AARCH64=c6g.large
```

Comment:
nitpick, but rather than declaring these just to assign to other variables, how about defining the values directly in the ARCH if/elif/else ?